frontend/src/modules/controllers/ValueModal.tsx

```diff
@@ -57,6 +101,15 @@ interface ContentProps {
   max?: number;
 }
 
+/**
+ * ValueModal
+ *
+ * @component A container for displaying value modal settings.
```

Comment:
What is the "value modal setting" which a `ValueModal` would display? What's the relationship, if any, between `ValueModal` and `SetValueContent`? Where would `ValueModal` be the appropriate component to use?

Reply:
`SetValueContent` is modified version of `ValueModal` to show the content of `ValueModal` along with some specific change for displaying it in MultiStepPopup.

`ValueModal` as per current UI pages, we cannot use. We can see its usage if `ValuesPage` is enabled in UI. Could navigate to /values to see its workings